### PR TITLE
[ADDITION] Add Zeth.mstyles

### PR DESCRIPTION
**Pull request by Mas4hmad**
Add variation for visual styles

### Preview:
Untitled

### CODE REVIEW

**Comment by Mas4hmad** [comment]
i use selawik font with different version because from ReactOS is shifted up little bit. i'll do upload with separate PR if you please. Thanks

File: zeth.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+Name = Zeth
+Version = 0.0.1
+License = Freeware
+Description = Zeth Visual Style for ReactOS
+Size = 116 KiB
+Category = 15
+URLDownload = https://github.com/Mas4hmad/ms-visual-styles/raw/master/ZETH.msstyles
+SHA1 = D06F5C1ADB15FB892F07583133D02E23AC3575C7
+CDPath = none
+SizeBytes = 119.658 bytes
```
Comment (learn-more) [inline zeth.txt:11]:
```suggestion
SizeBytes = 119658
```

File: zeth.txt [hunk 2]
```diff
@@ -0,0 +1,14 @@
+[Section]
+Name = Zeth
+Version = 0.0.1
+License = Freeware
+Description = Zeth Visual Style for ReactOS
+Size = 116 KiB
```
Comment (learn-more) [inline zeth.txt:6]:
```suggestion
```

File: zeth.txt [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+Category = 15
+URLDownload = https://github.com/Mas4hmad/ms-visual-styles/raw/master/ZETH.msstyles
+SHA1 = D06F5C1ADB15FB892F07583133D02E23AC3575C7
```
Comment (learn-more) [inline zeth.txt:9]: Are you sure this is the correct hash?

File: zeth.txt (hunk 2 above)
Comment (learn-more) [reply zeth.txt:6]: This is not needed anymore

File: zeth.txt (hunk 1 above)
Comment (learn-more) [reply zeth.txt:11]: This should just be a number, not any formatting

File: zeth.txt [hunk 4]
```diff
@@ -0,0 +1,14 @@
+[Section]
+Name = Zeth
+Version = 0.0.1
+License = Freeware
+Description = Zeth Visual Style for ReactOS
+Size = 116 KiB
+Category = 15
+URLDownload = https://github.com/Mas4hmad/ms-visual-styles/raw/master/ZETH.msstyles
```
Comment (learn-more) [inline zeth.txt:8]: Does ReactOS install it correctly like this?

**Comment by learn-more** [comment]
To test this, disable the option 'Update the list of accessible programs at start,
and drop the file in the local rapps dir (Documents and settings\username\rapps I think)

File: zeth.txt (hunk 3 above)
Comment (Mas4hmad) [reply zeth.txt:9]: oh, nope. i input that SHA1 before commit into repo. i don't know if it SHA1 is different. will do correct

File: zeth.txt (hunk 4 above)
Comment (Mas4hmad) [reply zeth.txt:8]: that's not installer, just copy it manually into theme folder.

File: zeth.txt (hunk 1 above)
Comment (Mas4hmad) [reply zeth.txt:11]: thanks, will do correctly

**Comment by Mas4hmad** [comment]
> 
> 
> To test this, ... .

i have test the text file download and it doing correctly. thanks for advising. wix fix this soon

File: zeth.txt (hunk 4 above)
Comment (learn-more) [reply zeth.txt:7]: In that case it would be a good idea to either create an installer, or add instructions in the description?

Comment (Mas4hmad) [reply zeth.txt:7]: for my first step, i would prefer add instruction. instead, create an installer is less possible regarding my lack of C language, i will make 'create an installer' if i ready.

Comment (petr-akhlamov) [reply zeth.txt:7]: I usual make installer via 7zip SFX Builder.
https://sourceforge.net/projects/s-zipsfxbuilder/
You can open my sfx (*.exe) in it and learn how to make own.
https://github.com/petr-akhlamov/ros-repo/raw/master/lunabeta.exe

Comment (learn-more) [reply zeth.txt:7]: All those installers ask the user where to install (and provide a correct default),
it's better to use f.e. winrar that allows to specify the target directory....

Comment (Mas4hmad) [reply zeth.txt:7]: thanks for the hint, will do update.
thank you so much before

**Comment by Mas4hmad** [comment]
> 
> 
> This url does not work for me...

it's weird. i have given correct url, but it can't download. but if i click link in description, it's download flawlessly.

**Comment by Mas4hmad** [comment]
the problem solved. i was use wrong OS that using Rapps (it's Windows XP!). Should be fixed now in ReactOS.
thanks

**Pull request merged**